plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/BasePinotIntegrationConnectorSmokeTest.java

```diff
@@ -1000,7 +1001,16 @@ public static Object of(
     @Override
     public void testShowCreateTable()
     {
-        assertQueryFails("SHOW CREATE TABLE region", "No PropertyMetadata for property: pinotColumnName");
+        assertThat((String) computeScalar("SHOW CREATE TABLE region"))
+                .matches(format(
```

Comment:
Can we use `isEqualTo` instead? I suppose we don't need to use a regular expression here.